### ISSUE TITLE
Error with X-Mollie-Client-Info header

### ISSUE THREAD

**Issue by it-can** [mollie/mollie-api-php#612]
## Specifications

  - API Version: v2.40.2

## Describe the issue
Getting this error on my docker machine

```
"Linux f0f489981e90 5.10.104-linuxkit #1 SMP Wed Mar 9 19:05:23 UTC 2022 x86_64" is not valid header value
```

seems to be triggered by this

https://github.com/mollie/mollie-api-php/blob/master/src/MollieApiClient.php#L535

...

**Comment by it-can** [comment]
seems to be a bug in Guzzle psr7 https://github.com/guzzle/psr7/issues/489

**Comment by holtkamp** [comment]
Seems to also occur when upgrading `guzzle/psr7` 1.8.3 => 1.8.4 

Here are the changes: https://github.com/guzzle/psr7/compare/1.8.3...1.8.4

Forcing composer to install 1.8.3 using `"guzzlehttp/psr7": "1.8.3",` resolves the problem. 

I suppose this is not a Mollie issue, but Mollie will probably get lot of failed payments due to this.

**Comment by GrahamCampbell** [comment]
This will be fixed in 1.8.5, 2.1.2, 2.2.1.
